src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -192,6 +191,15 @@ def _deploy_arm_template_core(resource_group_name, deployment_name, template_fil
     else:
         return smc.deployments.create_or_update(resource_group_name, deployment_name, properties)
 
+def _load_json(location):
+    from six.moves.urllib.request import urlopen #pylint: disable=import-error
+    try:
+        txt = urlopen(location).read()
+        json_content = json.loads(txt.decode())
+    except: #pylint: disable=bare-except
+        json_content = get_file_json(location)
+    return json_content
```

Comment:
for `mywebsite/mytemplate.json`, the `urlopen` will report errors like `unknown url type`,  so local file will be loaded, so we should be fine